paasta_tools/cli/cmds/status.py

```diff
@@ -912,11 +912,19 @@ def append_pod_status(pod_status, output: List[str]):
         ("Pod Name", "Host", "Phase", "Uptime")
     ]
     for pod in pod_status:
+        color_fn = (
+            PaastaColors.green
+            if pod["phase"] == "Running" and pod["container_state"] == "Running"
+            else PaastaColors.red
+            if pod["phase"] in ("Failed", "CrashLoopBackOff")
```

Comment:
Crashloopbackoff is `pod['container_state']`

Reply:
fixed in latest version

screenshot: https://yelp-shootie.appspot.com/shot/5689814631317504

also fixed headings for -v (removed Job ID): https://yelp-shootie.appspot.com/shot/5668874989600768